using-mobile-extensions/adobe-analytics-mobile-services/README.md

```diff
@@ -329,7 +329,9 @@ Using the following API does not increment page views.
 
 {% tabs %}
 {% tab title="Android" %}
-No set up required. In Android, the SDK handles push tracking to Analytics.
+In Android, the SDK handles push tracking to analytics and no additional set up is required.  
```

Comment:
On Android, the SDK handles push tracking to Analytics without any additional setup.